app/assets/javascripts/katello/smart_proxies/capsule_status.js

```diff
@@ -0,0 +1,28 @@
+function loadPulp() {
+    var item = $('#pulp-status'),
+        version = $('#pulp-version'),
+        url = item.data('url');
```

Comment:
can't we use data-ajax-url in similar fashion to the other ajax calls? /cc @tbrisker